src/main/java/nitezh/ministock/activities/ChartActivity.java

```diff
@@ -1,21 +1,35 @@
 package nitezh.ministock.activities;
 
 import android.app.Activity;
+import android.app.PendingIntent;
+import android.appwidget.AppWidgetManager;
```

Comment:
`import android.app.PendingIntent;
import android.appwidget.AppWidgetManager;
import android.content.Context;
import android.content.Intent;`

not used and should be removed

Reply:
fixed